main/res/values/strings.xml

```diff
@@ -854,7 +854,7 @@
     <string name="gpx_import_loading_caches">Loading caches from .gpx file</string>
     <string name="gpx_import_loading_waypoints">Loading waypoints file</string>
     <string name="gpx_import_store_static_maps">Storing static maps</string>
-    <string name="gpx_import_caches_imported">caches imported</string>
+    <string name="gpx_import_caches_imported"> %1$d caches imported from %2$s</string>
```

Comment:
Please create new resources (and keep the old ones there), since the `strings.xml` files are shared between the `release` and the development (`master`) branches.